salt/modules/junos.py

```diff
@@ -999,7 +1020,7 @@ def zeroize():
     ret = {}
     ret['out'] = True
     try:
-        conn.cli('request system zeroize')
+        conn.rpc.request_system_zeroize()
```

Comment:
can you checck if this rpc is there atleast on 14.x

Reply:
Removed the change from this PR. Will confirm from test and replace it in next PR if possible.